pgwatch2/sql/config_store/metric_definitions.sql

```diff
@@ -626,6 +626,87 @@ $sql$,
 '{"prometheus_gauge_columns": ["numbackends", "postmaster_uptime_s", "backup_duration_s", "checksum_last_failure_s"]}'
 );
 
+insert into pgwatch2.metric(m_name, m_pg_version_from, m_sql, m_sql_su, m_column_attrs)
```

Comment:
I'd like to finish the discussion on my questions above; This seems to be the same code, but for the config DB management approach.

Reply:
@markwort and about those "double" to "bigint" casts - it's mainly because it's nicer to look at integer numbers :D And also for those high level stats for troublesome or DBs of interest one really doesn't care about the fractions, they will become insignificant very quickly in real life. All storage backends can deal with floats also btw, "numeric" must be avoided though.